src/test/kotlin/com/atlassian/performance/tools/jiraactions/api/scenario/TestingScenario.kt

```diff
@@ -0,0 +1,76 @@
+package com.atlassian.performance.tools.jiraactions.api.scenario
+
+import com.atlassian.performance.tools.jiraactions.api.SET_UP
+import com.atlassian.performance.tools.jiraactions.api.SeededRandom
+import com.atlassian.performance.tools.jiraactions.api.WebJira
+import com.atlassian.performance.tools.jiraactions.api.action.*
+import com.atlassian.performance.tools.jiraactions.api.measure.ActionMeter
+import com.atlassian.performance.tools.jiraactions.api.memories.adaptive.AdaptiveIssueKeyMemory
+import com.atlassian.performance.tools.jiraactions.api.memories.adaptive.AdaptiveIssueMemory
+import com.atlassian.performance.tools.jiraactions.api.memories.adaptive.AdaptiveJqlMemory
+import com.atlassian.performance.tools.jiraactions.api.memories.adaptive.AdaptiveProjectMemory
+
+class TestingScenario : Scenario {
+
+    override fun getActions(jira: WebJira, seededRandom: SeededRandom, meter: ActionMeter): List<Action> {
```

Comment:
`TestingScenario.getActions` isn't called.
Let's remove it and simply run `jira.configureBackupPolicy().disable()`.

Previously I was considering merging ITs together, which would cover "backups vs actions" case for free. But we decided to split integration tests per feature, so the case is no longer worth it.

Reply:
I can not remove `getActions` because it is mandatory to override this function.
Do you propose to return null? What is the point in that case even if this function isn't called anywhere?

Reply:
I propose to remove `TestingScenario`

Reply:
Ok, so I will remove it and modify `JiraEditScenario` and add a similar class to `TestingSetup` to the `RichTextEditorIT` file.

Reply:
`RichTextEditorIT` tests edit actions, because they're impacted by `RichTextEditorTextArea` and `RichTextEditorConfiguration` - they use the RTE.
What do we need `JiraEditScenario` for here?

Reply:
Aha, I assumed wrongly that my test needs `Scenario` which is no true. I can fire `TestingSetup` action manually.